engine-rest/engine-rest-openapi/src/main/templates/paths/history/process-instance/get.ftl

```diff
@@ -53,7 +53,8 @@
                                     "superCaseInstanceId":null,
                                     "caseInstanceId":null,
                                     "tenantId":null,
-                                    "state":"ACTIVE"
+                                    "state":"ACTIVE",
```

Comment:
Validated in swagger editor
<details>
<summary>Screenshot</summary>
<img width="1417" alt="Screenshot 2024-09-09 at 4 23 59 PM" src="https://github.com/user-attachments/assets/dfa9e624-cb20-4186-bf6a-f9bf4eff786d">

</details>